### PR TITLE
fix: removing `bearer` from auth header when using azure

### PR DESCRIPTION
**Pull request by fredliang44**

### CODE REVIEW

File: app/api/common.ts [hunk 1]
```diff
@@ -9,7 +9,9 @@ const serverConfig = getServerSideConfig();
 export async function requestOpenai(req: NextRequest) {
   const controller = new AbortController();
 
-  const authValue = req.headers.get("Authorization") ?? "";
+  const authValue =
```
Comment (headphonejames) [inline app/api/common.ts:12]: fyi, this breaks the app for openai API - thanks!

**Comment by Cow258** [comment]
the auth broken after this update, 
**POST** - `/api/openai/v1/chat/completions`
401 Unauthorized

image
螢幕截圖 2023-12-25 08 41 24
image

**Comment by H0llyW00dzZ** [comment]
yeah it's so bad, it just giving a bug lmao

**Comment by wenwenwen888** [comment]
改动后报错了

**Comment by H0llyW00dzZ** [comment]
> 改动后报错了

https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/issues/3628#issuecomment-1868649693

**Pull request merged**